tests/scancode/test_cli.py

```diff
@@ -452,3 +452,25 @@ def test_scan_can_run_from_other_directory():
         print(stderr)
     assert rc == 0
     check_scan(test_env.get_test_loc(expected_file), result_file, strip_dates=True, regen=False)
+
```

Comment:
@pombredanne Yes I'll add them as well. Sorry for the inconvenience. Actually, I was confused, as to why the tests weren't passing, even though it seemed to work correctly when I executed the Scancode CLI locally though. That was my confusion actually.